scrapers/CopyToGallery.py

```diff
@@ -0,0 +1,124 @@
+import json
+import sys
+import os
+
+import py_common.graphql as graphql
+import py_common.log as log
+
+SERVER_IP = "http://localhost:9999"
+# API key (Settings > Configuration > Authentication)
+STASH_API = ""
+SERVER_URL = SERVER_IP + "/graphql"

```

Comment:
you forgot to remove this
the `graphql.callGraphQL` function gets those from `py_common/config.py` so they are not needed/used here